### PR TITLE
Updated instance types for shared monitoring

### LINKED ISSUE #22: change_shared_monitoring instance_type

### PR DESCRIPTION
**Pull request by pasipa2**
Closes #22

### CODE REVIEW

File: shared-monitoring/elasticsearch_ecs.tf [hunk 1]
```diff
@@ -153,25 +153,61 @@ data "template_file" "userdata_ecs" {
...
   }
 }
 
 ############################################
 # CREATE LAUNCH CONFIG FOR EC2 RUNNING SERVICES
 ############################################
 
-module "launch_cfg" {
-  source                      = "git::https://github.com/ministryofjustice/hmpps-terraform-modules.git?ref=master//modules//launch_configuration//blockdevice"
+module "launch_cfg_az1" {
```
Comment (jsbarbergit) [inline shared-monitoring/elasticsearch_ecs.tf]: Could switch to using a count loop to simplify this code and remove duplication/repetition. Example of something similar: https://github.com/ministryofjustice/hmpps-mis-terraform-repo/pull/108

File: shared-monitoring/variables.tf [hunk 2]
```diff
@@ -46,8 +46,8 @@ variable "es_service_desired_count" {
   default = 4
 }
 
-variable "es_ebs_volume_size" {
-  default = 200
+variable "es_block_device" {
+  default = "/dev/nvme1n1"
```
Comment (jsbarbergit) [inline shared-monitoring/variables.tf]: should also update the es_instance_type to be a nitro based type so this default block device will work, e.g. t3.*

File: shared-monitoring/elasticsearch_ecs.tf (hunk 1 above)
Comment (jsbarbergit) [inline shared-monitoring/elasticsearch_ecs.tf]: This code duplication/repetition could be removed by switching to a count loop. Something along the lines of this: https://github.com/ministryofjustice/hmpps-mis-terraform-repo/pull/108
Not 100% sure of the context, but a move to count based scaling could simplify scaling in different sized environments.

**Review by jsbarbergit — APPROVED** [review]

File: shared-monitoring/variables.tf (hunk 2 above)
Comment (pasipa2) [reply shared-monitoring/variables.tf]: changed

**Pull request merged**